client/src/components/ReportWorkflow.js

```diff
@@ -187,3 +188,39 @@ export const ReportCompactWorkflow = ({ workflow }) => (
 ReportCompactWorkflow.propTypes = {
   workflow: PropTypes.array.isRequired
 }
+
+export const CompactRowReportWorkflow = ({ workflow, compactStyle }) => (
+  <Fieldset
+    className="workflow-fieldset compact"
+    title="Workflow"
+    compactStyle={compactStyle}
+  >
+    <CompactRow
```

Comment:
`Fieldset` already displays the content within a `CompactRow`. Could it be possible to re-use existing `ReportCompactWorkflow` with a little tweak? Which may also create a possibility to get rid of `ROW_TYPES.onlyData` as it is only used in here.

Reply:
If some field is using a Fieldset, it means it has fields of fields ( I map Fieldset to table>tbody, only its children fields are rendered with CompactRow)
So conceptually:
`Fieldset(title, children_with_fields) maps to: CompactRow (th, td > table > tbody > each child tr > th = each_child_fieldname + td = each_child_content)`